save-preprocessor/src/main/kotlin/org/cqfn/save/preprocessor/controllers/DownloadProjectController.kt

```diff
@@ -230,10 +232,29 @@ class DownloadProjectController(private val configProperties: ConfigProperties,
                     log.error("Error to update test with url=$testSuiteUrl, path=$testSuitePaths")
                 }
                 .collect(Collectors.toList())
-                .subscribe()
-        }
+        }.collectList()
+            .flatMap {
+                deleteOldStandardTestSuites(newTestSuites)
+            }.subscribe()
```

Comment:
Did you try to return this flux instead of subscribing?

Reply:
Fixed